executor/index_lookup_join.go

```diff
@@ -473,6 +475,10 @@ func (iw *innerWorker) handleTask(ctx context.Context, task *lookUpJoinTask) err
 func (iw *innerWorker) constructLookupContent(task *lookUpJoinTask) ([]*indexJoinLookUpContent, error) {
 	lookUpContents := make([]*indexJoinLookUpContent, 0, task.outerResult.NumRows())
 	keyBuf := make([]byte, 0, 64)
+	keyTypes := make([]*types.FieldType, 0, len(iw.keyCols))
```

Comment:
move line#478~481 to line #500 ?

Reply:
line500 is in a for loop.

Reply:
before `for loop` and after `if`

Reply:
The `if` branch is already in a for loop